js/preferences/Preferences.js

```diff
@@ -201,6 +286,53 @@ export async function runtime_search_path_set(type, search_path) {
 
 }
 
+/**
+ * Set the users path for runtime of a given type.
+ * @param {RuntimeChannelType} type 
+ * @param {string} users_path 
+ */
+export async function users_search_path_set(type, users_path) {
+
+    prefs.runtime_opts.type_opts[type].users_path = users_path;
+    await save();
+
+    users[type] = null;
+
+    const res = await user_list_load_type(type);
+
+    if (!res.ok) {
+
+        const err = new Err(
+            `Failed to load ${type} user list`,
+            res.err,
+            'Make sure the users path is valid!'
```

Comment:
I appreciate leaving in the "how can you fix this" lines :P

Reply:
I really just copy-pasted the runtimes code for most of the user folder-related code.